### ISSUE TITLE
SFDX Source Pull can't retrieve picklist values

### ISSUE THREAD

**Issue by bdovh** [forcedotcom/cli#1264]
### Summary

We can’t retrieve the specified metadata object: PicklistValue. Certain metadata types, like PicklistValue1 are not currently supported by the CLI.
File a bug here: https://github.com/forcedotcom/cli/issues and provide the name of the unsupported metadata type
We can’t retrieve the specified metadata object: PicklistValue. Certain metadata types, like PicklistValue2 are not currently supported by the CLI.
File a bug here: https://github.com/forcedotcom/cli/issues and provide the name of the unsupported metadata type

### Steps To Reproduce:

**Repository to reproduce:** [dreamhouse-lwc](https://github.com/dreamhouseapp/dreamhouse-lwc)

Update picklist values , try to perform sfdx force:source:pull

***NOTE:** If your issue is not reproducable by dreamhouse-lwc, i.e. requires specific metadata or files, we **require** a link to a simple Salesforce project repository with a script to setup a scratch org that reproduces your problem.*

1.  This is step 1.
1.  This is step 2. All steps should start with '1.'

### Expected result

Expect picklist values to be pulled

### Actual result

Some error saying picklist values are not supported

### System Information

{
        "cliVersion": "sfdx-cli/7.123.0",
        "architecture": "darwin-x64",
        "nodeVersion": "node-v14.18.1",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.2.3 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-update 1.5.0 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "alias 1.1.22 (core)",
                "apex 0.3.0 (core)",
                "auth 1.7.3 (core)",
                "config 1.2.41 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.3 (core)",
                "evergreen 0.34.0",
                "└─ evergreen-build 0.20.2",
                "generator 1.2.0 (core)",
                "limits 1.2.2 (core)",
                "org 1.8.3 (core)",
                "salesforce-alm 53.0.0 (core)",
                "schema 1.0.10 (core)",
                "sfdx-cli 7.123.0 (core)",
                "shane-sfdx-plugins 4.43.0",
                "├─ @mshanemc/plugin-streaming 1.1.7",
                "└─ @mshanemc/sfdx-sosl 1.1.0",
                "source 1.2.4 (core)",
                "telemetry 1.2.8 (core)",
                "templates 52.4.0 (core)",
                "trust 1.0.8 (core)",
                "user 1.5.2 (core)"
        ],
        "osVersion": "Darwin 20.5.0"
}

- If you are using `sfdx`
	- Run `sfdx version --verbose --json`
- If you are using `sf`
	- Run `sf version` **AND** `sf plugins --core`
- Paste the output here
### Additional information

_Feel free to attach a screenshot_.

**Comment by WillieRuemmele** [comment]
The `PickList` metadata is actually stored as a [StandardValueSet](https://salesforce.stackexchange.com/questions/215236/move-standard-picklist-values-using-sfdx). Using `dreamhouse-lwc` I was able to `force:source:pull` that metadata type 
```
 ➜  sfdx force:source:pull   
=== Pulled Source
STATE  FULL NAME          TYPE              PROJECT PATH
─────  ─────────────────  ────────────────  ─────────────────────────────────────────────────────────────────────────────
Add    test               GlobalValueSet    force-app/main/default/globalValueSets/test.globalValueSet-meta.xml
Add    LeadSource         StandardValueSet  force-app/main/default/standardValueSets/LeadSource.standardValueSet-meta.xml
```

could you please provide more detailed steps to reproduce it, or you can try using the `source:beta:*` commands available in `7.125.0`

**Comment by bdovh** [comment]
I think the issue reproduces when you create picklist first, then you pull, then you add more picklist values or activate picklist values from the list and pull again. The second pull fails

**Comment by WillieRuemmele** [comment]
could you try using the new `source:beta:pull` command? This bug might already be fixed in there, and if it isn't we'll fix it in there.

**Comment by bdovh** [comment]
> could you try using the new `source:beta:pull` command? This bug might already be fixed in there, and if it isn't we'll fix it in there.

Interesting idea, is that available in my sfdx?

**Comment by bdovh** [comment]
Warning: source:beta:pull is not a sfdx command.

**Comment by WillieRuemmele** [comment]
sorry @bdovh - it's available in `latest-rc: 7.125.0` it will be available in `7.125.0` as `latest` later today

**Comment by bdovh** [comment]
why this issue is closed?

**Comment by mshanemc** [comment]
this works in the new `beta:pull.`. There was a bug pulling CustomFields that's now solved in 7.130.1 (currently RC)

**Comment by mshanemc** [comment]
closed with new commands GA
